packages/crayons-core/src/components/country-phone/country-phone.tsx

```diff
@@ -0,0 +1,395 @@
+import {
+  Component,
+  Prop,
+  h,
+  State,
+  Element,
+  Event,
+  EventEmitter,
+  Watch,
+  Method,
+} from '@stencil/core';
+import { renderHiddenField } from '../../utils';
+
+import countries from './countries.json';
+
+import { isValidPhoneNumber, parsePhoneNumber } from 'libphonenumber-js';
+
+import { CountryCode } from 'libphonenumber-js/types';
+
+@Component({
+  tag: 'fw-country-phone',
+  styleUrl: 'country-phone.scss',
+  shadow: true,
+})
+export class CountryPhone {
+  @Element() host!: HTMLElement;
+
+  /**
+   * Phone input
+   */
+  @State() phoneNumber: string;
+
+  /**
+   * Helps to stop unwanted rendering when value is updated from inside the component
+   */
+  @State() isValueUpdatedFromInside = false;
```

Comment:
Kindly consider making this a private variable.

Reply:
Fixed ✅